docs/guide/macOS.md

```diff
@@ -5,14 +5,26 @@ Use `macos_instance` in your `.cirrus.yml` files:
 
 ```yaml
 macos_instance:
-  image: big-sur-base
+  image: monterey-base
 
 task:
   script: echo "Hello World from macOS!"
 ```
 
 ### List of available images
 
+#### macOS Monterey
+
+* `monterey-base` - vanilla macOS with Brew and Command Line Tools pre-installed.
+* `monterey-xcode-NN` - based of `catalina-base` with Xcode NN and couple other packages pre-installed: 
+  `cocoapods`, `fastlane`, `rake` and `xctool`. [Flutter](https://flutter.dev/) and Android SDK/NDK are also pre-installed.**
```

Comment:
```suggestion
  `cocoapods`, `fastlane`, `rake` and `xctool`. [Flutter](https://flutter.dev/) and Android SDK/NDK are also pre-installed.
```

Reply:
The same thing also exists in the "macOS Big Sur" section for some reason.

Reply:
That's weird. See a2edaa5

Reply:
Added in 14a4302